### DISCUSSION TITLE
Maximum Repository Size?

### DISCUSSION THREAD

**Discussion by DomiMartinGlogi** [community/community#120943] [Repositories] [answered]
### Select Topic Area

Question

### Body

Is there any limitation as to how large a single GitHub Repository can be, and are there any differences in that regard for Teams/Enterprise Organisation or Pro-Users compared to non-paying users? I know that there is a limit to how large a singular file can be to be uploaded to GitHub.

**Comment by fdistasio** [comment, marked as answer]
The difference between the various github plans refers only to the maximum size of individual files. As regards the maximum repository size, there is no fixed one but the recommendation is to stay under 5GB.

https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github
https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-git-large-file-storage

**Comment by DomiMartinGlogi** [comment]
Thank you very much!
